DevelopGitHub/GitHubExtension/GitHubExtension.Security.Tests/Mocks/MockForEnumerableQuery.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.Entity.Infrastructure;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GitHubExtension.Security.Tests.Mocks
+{
+    class MockForEnumerableQuery<T> : EnumerableQuery<T>, IDbAsyncQueryProvider
```

Comment:
Btw, can you verify this one class EnumerableQuery(namespace System.Linq)...May be it can replace this specific class, but I am not sure.